docs/content/users/usage/commands.md

```diff
@@ -716,6 +717,9 @@ ddev list
 
 # List all running projects
 ddev list --active-only
+
+# List all wordpress
```

Comment:
Minor nitpick to clarify the product name (proper noun) and add “projects” for consistency with the comment above:

```suggestion
# List all WordPress projects
```

Reply:
Absolutly. Good finding.